images/linux/scripts/helpers/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+################################################################################
+##  File:  install-helpers.sh
```

Comment:
nitpick: "install.sh" is not clear file name and I cannot understand what I can expect there, I would suggest to rename file to "os.sh" if it makes sense to you.

Reply:
@dsame I agree with @sergey-akhalkov, the script name is not clear.